jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/advanced/AdvancedFiltersPresenter.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.client.filters.advanced;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import org.jbpm.workbench.common.client.filters.saved.SavedFilterSelectedEvent;
+import org.jbpm.workbench.common.client.resources.i18n.Constants;
+import org.jbpm.workbench.df.client.filter.AdvancedFilterEditor;
+import org.jbpm.workbench.df.client.filter.FilterSettingsManager;
+import org.jbpm.workbench.df.client.filter.SavedFilter;
+import org.uberfire.client.annotations.WorkbenchPartTitle;
+import org.uberfire.client.annotations.WorkbenchPartView;
+import org.uberfire.lifecycle.OnOpen;
+
+public abstract class AdvancedFiltersPresenter {
+
+    private final Constants constants = Constants.INSTANCE;
+
+    protected FilterSettingsManager filterSettingsManager;
+
+    AdvancedFilterEditor advancedFilterEditorView;
+
+    private Event<SavedFilterSelectedEvent> savedFilterSelectedEvent;
+
+    @WorkbenchPartView
+    public IsWidget getView() {
+        return advancedFilterEditorView;
+    }
+
+    @WorkbenchPartTitle
+    public String getTitle() {
+        return constants.AdvancedFilters();
+    }
+
+    @PostConstruct
+    public void init() {
+        initAdvancedFilters();
```

Comment:
Is it needed to execute _initAdvancedFilters_ on both _PostConstruct_ and _OnOpen_ methods?